deploy/crds/mdc_v1alpha1_mobiledeveloperconsole_crd.yaml

```diff
@@ -34,6 +34,18 @@ spec:
         metadata:
           type: object
         spec:
+          properties:
+            oAuthClientId:
+              description: OAuthClientId is the id of the OAuthClient that the OAuthProxy
+                for this resource uses.
+              type: string
+            oAuthClientSecret:
+              description: OAuthClientId is the secret of the OAuthClient that the
+                OAuthProxy for this resource uses.
+              type: string
+          required:
+          - oAuthClientId
```

Comment:
Slight preference to de-capitalize the "Auth" part here (`oauthClientId` rather than `oAuthClientId`), but that's just a personal opinion and not necessarily more or less correct, and it really doesn't matter either way :see_no_evil:

Reply:
I did it like that initially. Operator SDK complains in that case, or one of the things it uses! Hard to believe it but true.
Took me 30 mins to find out that it didn't like the naming...

See https://github.com/operator-framework/operator-sdk/issues/1222

Reply:
Interesting!